charts/govuk-rails-app/templates/nginx-configmap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-nginx-conf
+data:
+  nginx.conf: |-
```

Comment:
Something for later, but we should validate the contents of nginx.conf at some stage - do we need to change the defaults for `client_max_body_size` and similar, configure log levels or output format, etc.

Reply:
I agree - i believe there is a Nginx related Trello card - i will make a note: https://trello.com/c/Lkb8N313/381-epic-nginx-and-varnish-changes-needed